packages/react-data-grid/src/HeaderRow.tsx

```diff
@@ -147,20 +134,17 @@ export default class HeaderRow<R> extends React.Component<HeaderRowProps<R>> {
   }
 
   render() {
-    const cellsStyle: React.CSSProperties = {
-      width: this.props.width ? this.props.width + getScrollbarSize() : '100%',
-      height: this.props.height
-    };
+    const { height, rowType } = this.props;
 
-    // FIXME: do we need 2 wrapping divs?
     return (
       <div
-        style={this.props.style}
+        ref={this.headerRow}
+        style={{
+          height: rowType === HeaderRowType.FILTER ? 500 : height
```

Comment:
![image](https://user-images.githubusercontent.com/567105/65599404-aef62980-df95-11e9-8cc1-772faa3bea3c.png)
❓

Reply:
This is to show the dropdown filters on the header row otherwise they are cut off. It is an old behavior so I did not fix it.